src/map/skill.c

```diff
@@ -5869,14 +5880,10 @@ static int skill_castend_id(int tid, int64 tick, int id, intptr_t data)
 		}
 		if (target && target->m == src->m) {
 			//Move character to target anyway.
-			int dir, x, y;
-			dir = map->calc_dir(src,target->x,target->y);
-			if( dir > 0 && dir < 4) x = -2;
-			else if( dir > 4 ) x = 2;
-			else x = 0;
-			if( dir > 2 && dir < 6 ) y = -2;
-			else if( dir == 7 || dir < 2 ) y = 2;
-			else y = 0;
+			enum unit_dir dir = map->calc_dir(src,target->x,target->y);
```

Comment:
Missing spaces in argument list: `map->calc_dir(src, target->x, target->y)`

Reply:
done.